sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -169,16 +195,25 @@ public partial class ContainerRepository
     {
         protected ContainerRepository() { }
         public virtual string Name { get { throw null; } }
+        public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
         public virtual System.Uri RegistryEndpoint { get { throw null; } }
+        public virtual Azure.Response Delete(string name, Azure.RequestContext context = null) { throw null; }
```

Comment:
Remove `name`, and require `RequestContext` - we can do this manually, but we should make a note on the DPG issue tracking the work.